server/views/pages/courtCaseDetails.njk

```diff
@@ -80,57 +84,64 @@
             <ul class="offence-card-actions-list">
                 <li class="offence-card-action-link">
 
-                            {% if draft == 'true' %}
-                            <a class="govuk-link--no-visited-state" href="/person/{{ nomsId }}/add-court-case/{{ courtCaseDetails.courtCaseUuid }}/add-court-appearance/{{ appearance.appearanceUuid }}/resume-draft">
+                    {% if draft == 'true' %}
```

Comment:
mainly just ide format changes but Ichanged the if / else here to make it a bit more readable and remove a code smell